examples/init_containers/Makefile

```diff
@@ -1,5 +1,6 @@
-IMAGE_NAME=seldonio/rclone-init-container
-IMAGE_VERSION := $(shell cat ../../version.txt)
+IMAGE_NAME=seldonio/rclone-init-container-example
+# IMAGE_VERSION := $(shell cat ../../version.txt)
+IMAGE_VERSION=0.1
```

Comment:
Why is image version not based on release. Is it assumed it is unlikely to change be effected by release. If so maybe remove the commented line?

Reply:
This is only an example that actually has no dependency on the in-repo code. It just changes the `entrypoint` of the upstream image:
```
FROM rclone/rclone:latest
ENTRYPOINT ["rclone", "copy"]
```

When later we will add a proper rclone-based alternative to storage.py (not just an example of custom init container) then I guess it will make sense to version it with `../../version.txt`.